server/services/store/sqlstore/migrations/migrations_files/000001_init.up.sql

```diff
@@ -12,4 +12,4 @@ CREATE TABLE IF NOT EXISTS {{.prefix}}blocks (
 	update_at BIGINT,
 	delete_at BIGINT,
 	PRIMARY KEY (id, insert_at)
-){{if .mysql}}CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci{{end}};
+) {{if .mysql}}DEFAULT CHARACTER SET utf8mb4{{end}};
```

Comment:
Nice. I'm not 100% sure if this solves the problem, but looks like an acceptable solution.

Reply:
It did work and I tested on MySQL 8.7 and the 15.